engine/app/components/citizens_advice_components/header.html.haml

```diff
@@ -0,0 +1,43 @@
+%header.cads-header
+  .cads-grid-container
+    .cads-grid-row
+      - if skip_links.present?
+        %nav.cads-skip-links
+          - skip_links.each do |link|
+            = link_to(link.title, link.url, class: "cads-skip-links__link")
+
+      .cads-grid-col-md-2.cads-header__logo-row
+        %a.cads-logo{ href: homepage_url, title: t("citizens_advice_components.shared.logo_title") }
+        %button.cads-header__search-reveal.js-cads-search-reveal.cads-icon_search{ toggle_button_attributes }
+      .cads-grid-col-md-10.cads-header__search-row
+        %ul.cads-header__links.js-cads-copy-into-nav
```

Comment:
Wondering if it makes sense to bundle the global navigation in with this component. Does it make sense for them to be used separately?